src/store/modules/order/actions.ts

```diff
@@ -33,6 +37,14 @@ const actions: ActionTree<OrderState, RootState> = {
     }
     return resp;
   },
+  async updateProductCount({ commit, state }, payload ) {
+    state.current.items.forEach((item: any) => {
```

Comment:
I think we should use find and update the qunatity

Reply:
Updated the code to use find and update the quantity.